demos/ASP.Net Core/Divergent.Frontend/Controllers/api/ValuesController.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+
+namespace Divergent.Frontend.Controllers.api
+{
+    [Produces("application/json")]
+    [Route("api/Values")]
+    public class ValuesController : Controller
+    {
+        // GET: api/Values
+        [HttpGet]
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET: api/Values/5
+        [HttpGet("{id}", Name = "Get")]
+        public string Get(int id)
+        {
+            return "value";
+        }
+        
+        // POST: api/Values
+        [HttpPost]
+        public void Post([FromBody]string value)
+        {
+        }
+        
+        // PUT: api/Values/5
+        [HttpPut("{id}")]
+        public void Put(int id, [FromBody]string value)
+        {
+        }
+        
+        // DELETE: api/ApiWithActions/5
+        [HttpDelete("{id}")]
+        public void Delete(int id)
+        {
+        }
+    }
+}
```

Comment:
it's just there to prove that we don't break existing functionalities by injecting ourselves into the routing pipeline. We can safely remove it.